alpine/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM puppet/puppet-agent-alpine
+
+RUN apk add --update git \
+    && git clone https://github.com/lingua-pupuli/puppet-editor-services.git
+
+ENTRYPOINT ["/usr/bin/ruby"]
+CMD ["/puppet-editor-services/puppet-languageserver", "--ip=0.0.0.0"]
```

Comment:
Suggest `CMD ["/puppet-editor-services/puppet-languageserver", "--ip=127.0.0.1", "--port=8082", "--timeout=0"] `

Reply:
Better with `--ip=0.0.0.0`.  There are some issues with using localhost in docker-for-windows

Reply:
This is fairly easy to update, will do.

Reply:
Shrug. Worked for me on my machine*

*Doesn't know what he is doing with docker